tests/Assetic/Test/Filter/ScssphpFilterTest.php

```diff
@@ -86,10 +86,10 @@ public function testCompassExtensionCanBeEnabled()
 
     public function testCompassExtensionCanBeDisabled()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             "Exception",
-            "Undefined mixin box-shadow: failed at `@include box-shadow(10px "
-            ."10px 8px red);` line: 4"
+            preg_quote("/Undefined mixin box-shadow: failed at `@include box-shadow(10px 10px 8px red);`").
+            ".*?".preg_quote(" line 4/")
```

Comment:
I suggest writing the regex directly instead of building it with multiple preg_quote